modules/cluster-logging-release-notes-5.4.z.adoc

```diff
@@ -0,0 +1,26 @@
+
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+[id="cluster-logging-release-notes-5-4-z"]
+= {logging-title-uc} 5.4.z
+The following advisories are available for {logging} 5.4.z
```

Comment:
```suggestion
The following advisories are available for {logging} 5.4.z:
```

Reply:
Question: Is this a template? Just curious.

Reply:
Yes.